src/Matrix.jl

```diff
@@ -6817,6 +6817,26 @@ function matrix(R::NCRing, arr::AbstractMatrix{T}) where {T}
    end
 end
 
+function matrix(arr::AbstractMatrix{T}) where {T<:NCRingElem}
+   r, c = size(arr)
+   (r < 0 || c < 0) && error("Array must be non-empty")
+   R = parent(arr[1, 1])
+   all(e -> parent(e) === R, arr) || errpr("Non-compatible elements")
```

Comment:
Typo. So clearly this case is never tested ;-)
```suggestion
   all(e -> parent(e) === R, arr) || error("Non-compatible elements")
```

Reply:
Uff :)